Portland 2021 speakers (#1436)
writethedocs/www: docs/conf/portland/2021/news/announcing-speakers.rst

:template: {{year}}/generic.html

.. post:: February 14, 2021
   :tags: portland-2021, speakers, tickets, visiting, sponsors

Announcing speakers and sessions
================================

> We're writing today with an update on the fast-approaching Portland conference!

Reviewer: ```suggestion
We're writing today with an update on the fast-approaching Portland conference.
```
Maybe not have every sentence end with an exclamation mark here.

Author: Sounds good!

> First off, we're delighted to announce that we've finalized our speakers for the event!

Reviewer 2: ```suggestion
First off, we're delighted to announce that we've finalized our speakers for the event.
```

Full speaker line-up
--------------------

Every year, we look to bring a wide range of voices to the Write the Docs stage. Because the role of "documentarian" looks so different to each of us, we aim for a line-up that spans a good mix of practical, philosophical, and technical topics – with the odd whimsical one thrown in for good measure.
We're really excited about the presentations we've got this year – we hope you are too!

.. datatemplate::
   :source: /_data/{{shortcode}}-{{year}}-sessions.yaml
   :template: {{year}}/speakers-simple-list.rst

Head over to the `Speakers page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/speakers/>`_ to see the full abstracts.
We hope there’s something there for everyone to enjoy, and also something that will broaden your horizons a bit.

Tickets are going fast!
-----------------------

If you don't have your ticket yet, now's a great time to snap one up. Conference tickets have sold out for the last several years, and they go faster along with the speaker announcement, so if you're planning to come, don't leave it to the last minute.

`Buy your tickets here! <https://www.writethedocs.org/conf/portland/{{year}}/tickets/>`_

> 

Reviewer 2: I'd add the copy about t-shirt sales here (we should have the text in last year's post), and also turn on the t-shirt flag in the yaml file.

Link to this year's shirts (notice the URL format is different in case it matters):
https://teespring.com/wtd-pdx-2021

> 

Reviewer 2: ```suggestion
Shirts on sale
--------------

The `Write the Docs Portland 2021 Pop-Up Shop <https://teespring.com/wtd-pdx-2021>`_ is open, and starting year we updated the logo to include the year of the conference! You can order loose and fitted options from different fabrics, and you can choose exactly the size and cut you want.

The campaign will run in 3-day batches until May 31st. Shipping is available from the USA or Europe. However, due to the global shipping situation, we recommend you order your shirt at least 2-3 weeks in advance if you want to sport your new threads during the conference.

```

:template: {{year}}/generic.html

> .. post:: February 14, 2021

Reviewer 3: Need to update this before publishing.

> .. post:: February 14, 2021

Author: ```suggestion
.. post:: February 17, 2021
```

   :tags: portland-2021, speakers, tickets, visiting, sponsors

Announcing speakers and sessions
================================

We're writing today with an update on the fast-approaching Portland conference.

> First off, we're delighted to announce that we've finalized our speakers for the event.

Reviewer 3: We aren't doing anything other than this first, so feels like a weird transition. Also probably good to mention the other new items in the head. 

```suggestion
We're delighted to announce that we've finalized our speakers for the event and have t-shirts on sale.
```

Full speaker line-up
--------------------

Every year, we look to bring a wide range of voices to the Write the Docs stage. Because the role of "documentarian" looks so different to each of us, we aim for a line-up that spans a good mix of practical, philosophical, and technical topics – with the odd whimsical one thrown in for good measure.
We're really excited about the presentations we've got this year – we hope you are too!

.. datatemplate::
   :source: /_data/{{shortcode}}-{{year}}-sessions.yaml
   :template: {{year}}/speakers-simple-list.rst

Head over to the `Speakers page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/speakers/>`_ to see the full abstracts.
We hope there’s something there for everyone to enjoy, and also something that will broaden your horizons a bit.

Tickets are going fast!
-----------------------

If you don't have your ticket yet, now's a great time to snap one up. Conference tickets have sold out for the last several years, and they go faster along with the speaker announcement, so if you're planning to come, don't leave it to the last minute.

`Buy your tickets here! <https://www.writethedocs.org/conf/portland/{{year}}/tickets/>`_

> Shirts on sale

Reviewer 3: I'd put this above the ticket callout, since its new.

--------------

The `Write the Docs Portland 2021 Pop-Up Shop <https://teespring.com/wtd-pdx-2021>`_ is open, and starting year we updated the logo to include the year of the conference! You can order loose and fitted options from different fabrics, and you can choose exactly the size and cut you want.

The campaign will run in 3-day batches until May 31st. Shipping is available from the USA or Europe. However, due to the global shipping situation, we recommend you order your shirt at least 2-3 weeks in advance if you want to sport your new threads during the conference.

Thanks to our sponsors
----------------------

We are so grateful to have our sponsors help in bringing these events to life every year. Thanks sincerely to the following companies for supporting the Write the Docs community:

.. datatemplate::
   :source: /_data/{{shortcode}}-{{year}}-config.yaml
   :template: {{year}}/sponsors-simplelist.rst

> Head over to our `sponsorship prospectus <https://www.writethedocs.org/conf/portland/2020/sponsors/prospectus/>`_ for details on how your company can get involved!

Reviewer 3: Old prospectus. 
```suggestion
Head over to our `sponsorship prospectus <https://www.writethedocs.org/conf/portland/2021/sponsors/prospectus/>`_ for details on how your company can get involved!
```
